src/docfx/publish/PublishItem.cs

```diff
@@ -33,7 +33,13 @@ internal class PublishItem
         [JsonExtensionData]
         public JObject? ExtensionData { get; set; }
 
-        public PublishItem(string url, string? path, string? sourcePath, string locale, string[] monikers, string? configMonikerRange)
+        [JsonIgnore]
+        public ContentType ContentType { get; }
+
+        [JsonIgnore]
+        public string? Mime { get; }
```

Comment:
it would be great if we have one filed to present this publish page type like "conceptual", "toc", just like the `layout` defined in the DHS.